analyzer-comments/elm/ticket-please/destructure_assignTicketTo_argument.md

```diff
@@ -0,0 +1,3 @@
+# destructure assignTicketTo argument
+
```

Comment:
This is only tangential, but I don't think there is any reference to the `as` syntax in the concept or exercise docs. I had a google and this cheatsheet was the best source I could find.
https://gist.github.com/yang-wei/4f563fbf81ff843e8b1e#record
Maybe we should add it here, and to the exercise and concept docs, what do you think?

Reply:
We use it in the last example in the introduction of the concept, and we use this link: https://github.com/izdi/elm-cheat-sheet#operators 
Both those links are a bit shabby, so I like the idea of contributing to some docs, but which?

Reply:
I guess we can initially explain it will on exercism, in the concept, where we have full control and can do everything.
Those links are shabby, maybe we could put a shout out on the Elm slack? It's possible that any PR to the main slack won't get merged, but maybe elm-programming or something like that would like it ...